### DISCUSSION TITLE
New ServiceClaim CRD Name

### DISCUSSION THREAD

**Discussion by filariow** [primaza/primaza#292] [Polls]
As described in #275, we need a different CRD for ServiceClaims in Control Plane and in Application Namespaces.

**Comment by dperaza4dustbit** [comment]
I would prefer something other than Claim for ControlPlane Would be nice to have just a prefix like ControlPlaneServiceClaim

**Comment by filariow** [comment]
After discussion the winner is `Control Plane: ControlPlaneServiceClaim | Application Namespace: ApplicationServiceClaim`
